GiTinder/Services/ProfileService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using GiTinder.Data;
+using GiTinder.Models;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GiTinder.Services
+{
+    public class ProfileService
+    {
+        private GiTinderContext _context;
+
+        public ProfileService(GiTinderContext context)
+        {
+            _context = context;
+        }
+        public List<User> ReturnNProfilesRandomly(string username, int numberOfProfiles)
```

Comment:
@Riceqrisp let's call this `GetProfilesForUser`

Reply:
Changed